src/libguestfish.sh

```diff
@@ -30,6 +44,7 @@ coreos_gf_launch() {
 
 _coreos_gf_cleanup () {
     guestfish --remote -- exit >/dev/null 2>&1 ||:
+    rm "${tmpd}" -rf
```

Comment:
Hmm, `tmpd` might be a little too generic. Maybe something like `tmp_qemu_wrapper`?

Reply:
Done, and thanks for all the review =)